packages/components/src/select-control/stories/index.js

```diff
@@ -0,0 +1,59 @@
+/**
+ * External dependencies
+ */
+import { boolean, object, text } from '@storybook/addon-knobs';
+
+/**
+ * WordPress dependencies
+ */
+import { useState } from '@wordpress/element';
+
+/**
+ * Internal dependencies
+ */
+import SelectControl from '../';
+
+export default {
+	title: 'Components/SelectControl',
+	component: SelectControl,
+};
+
+const SelectControlWithState = ( props ) => {
+	const [ selection, setSelection ] = useState();
+
+	return (
+		<SelectControl
+			{ ...props }
+			value={ selection }
+			onChange={ setSelection }
+		/>
+	);
+};
+
+export const _default = () => {
+	const label = text( 'Label', 'Label Text' );
+	const hideLabelFromVision = boolean( 'Hide Label From Vision', false );
+	const help = text(
+		'Help Text',
+		'Help text to explain the select control.'
+	);
+	const multiple = boolean( 'Allow Multiple Selection', false );
+	const options = object( 'Options', [
```

Comment:
@brentswisher Haii!!! Thank you for adding all of these knobs! You've done a wonderful job with these Storybooks PRs 🙏 

For select based content, Storybook's Knob plugin has a special `select` function:
https://github.com/storybookjs/storybook/tree/master/addons/knobs#select

It's handy as it renders a `select` UI within the knobs area, making it easier to toggle between options.

I would recommended refactoring to use `select` instead of `object` :)

Reply:
@ItsJonQ thanks for all the reviews! In this case, the knob is for the options array that is sent to the `SelectControl` Component. Since that is expecting an array of objects, I think it needs to stay an object knob? 

The select knob is for rendering a `<select>` as the knob, here the select is the component in the story.

Reply:
Oh, that is true!! My apologies. I was reviewing it from the perspective of making it easier to poke at within Storybook. But you are right, the data structure of `select` isn't directly compatible with the `SelectControls` options.